pkg/platform/platform.go

```diff
@@ -275,3 +277,15 @@ func consolePluginCRDExists(cl cnoclient.Client) (bool, error) {
 	}
 	return true, nil
 }
+
+func IsUserDefinedIPsecMachineConfig(machineConfig *mcfgv1.MachineConfig) bool {
```

Comment:
NIT: I think this could use a docblock explaining what does a UserDefineIPsecMachineConfig cause.